aboutus.html

```diff
@@ -1 +1,39 @@
-hi
+<!DOCTYPE html>

```

Comment:
Maybe separate to be a section of each member or create each page for each member. But I think create section is better.